app/src/main/java/com/ucmobiledevelopment/freeways/MainActivity.kt

```diff
@@ -237,7 +239,7 @@ class MainActivity : ComponentActivity() {
         resultsMap ->
         var permissionGranted = false
         resultsMap.forEach {
-            if (it.value == true) {
+            if (it.value) {
```

Comment:
A good suggestion, though it appeared in a previous review.  When that happens, first in wins.